mmf/datasets/builders/visual_entailment/dataset.py

```diff
@@ -41,8 +41,14 @@ def load_item(self, idx):
                     features["image_info_0"]
                 )
             current_sample.update(features)
+        else:
+            image_path = sample_info["Flikr30kID"]
+            current_sample.image = self.image_db.from_path(image_path)["images"][0]
 
         label = LABEL_TO_INT_MAPPING[sample_info["gold_label"]]
         current_sample.targets = torch.tensor(label, dtype=torch.long)
 
         return current_sample
+
+    def format_for_prediction(self, report):
+        return []
```

Comment:
Do we need `format_for_prediction` for this dataset if we are not returning anything?

Reply:
The `VisualEntailmentDataset` is based on `VQA2Dataset` and inherited its `format_for_prediction` (that is incompatible), causing crash when running prediction report generation on UniT (which is sometimes needed in the new evaluation loop). This change restores the default `format_for_prediction` in `BaseDataset`.

(Also, I think generating an empty report is still better than directly crashing for `VisualEntailmentDataset`.)